cluster/saltbase/salt/kubelet/init.sls

```diff
@@ -31,19 +31,6 @@
     - mode: 400
     - makedirs: true
 
-#
```

Comment:
Was there a problem keeping /var/lib/kubelet/kubernetes_auth in addition to /var/lib/kubelet/kubeconfig?